Added Work And About Sections

gisachris opened the pull request:
in this PR i added the following:
added a works section
added an about section
added fonts required to project

gisachris committed:
Create MIT.md

gisachris committed:
Update MIT.md

gisachris committed:
Merge pull request #1 from gisachris/new-features

portfolio: setup and mobile first

ouasamine reviewed (commented):
Hi @gisachris 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is not documented in a professional way.
  - Your PR title is not descriptive: it should summarise the changes in one sentence.
  - Your branch name is not descriptive, it should have the name of the introduced feature eg "create-sidebar",  you will need to create a new branch with a proper name based on this branch, checkout to it and create a new PR or [change the base branch of this PR using this guide](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-base-branch-of-a-pull-request).

**Friendly reminder: You should make sure that your repository looks professional before review. Please take a step back apply 5 simple rules from this [lesson](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).**

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

Lookingfor00 commented on index.html:
```diff
@@ -43,6 +43,179 @@ <h1 class="text1">Hey There.<br>I'm Chris.</h1>
-
+    <div class="page-2-container">
+      <header class="heading">
+        <a href="index.html" class=""><img src="Title-top.png" alt="title" class=""></a> 
```
- [ ] Please notice that the title of the work section should be in a `h2` tag and should not be a image (`img`). this will make your code much more maintainable 😄 (specialy when you will want to change the title)

Lookingfor00 commented on index.html:
```diff
@@ -43,6 +43,179 @@ <h1 class="text1">Hey There.<br>I'm Chris.</h1>
+            </h2>
+            <p class="grid-item grid-item-3">
+              <img src="p-text.png" alt="paragraph" class="p-text">
+            </p>
```
- [ ] Please notice that the description of the work element should be in a `p` (paragraph) tag and should not be a image (`img`). this will make your code much more maintainable 😄 (specialy when you will want to change the description)

Lookingfor00 commented on index.html:
```diff
@@ -43,6 +43,179 @@ <h1 class="text1">Hey There.<br>I'm Chris.</h1>
+                  <section class="background">
+                  <h2 class="sec-grid-item-1 text-1">
+                    Profesional Art <br> Printing Data
+                  </h2>
```
- [ ] Kindly update `grid-container` and `sec-grid-container` to use flexbox instead of grid. The elements of the work section should use a flex-box to display their content. This will make your project way more maintainable as fex-box is an incredible 💪 😮 tool for responsiveness 😄

Lookingfor00 commented on index.html:
```diff
@@ -43,6 +43,179 @@ <h1 class="text1">Hey There.<br>I'm Chris.</h1>
+          <img src="line-top.svg" alt="line" class="line-top">
+        </nav>
+      </header>
+        <main class="body-container">
+          <div class="grid-container">
+            <img src="img-place-1.png" alt="portfolio-1" class="grid-item grid-item-1">
```
- [ ] Kindly, the main container of the work section (`<main class="body-container">`) should use grid to display is elements. This will make your project way more maintainable as grid is an incredible 💪 😮 tool for responsiveness with list of elements😄

Lookingfor00 commented on index.html:
```diff
@@ -43,6 +43,179 @@ <h1 class="text1">Hey There.<br>I'm Chris.</h1>
+            <img src="img-place-1.png" alt="portfolio-1" class="grid-item grid-item-1">
+            <h2 class="grid-item grid-item-2">
+              <img src="p-title-1.png" alt="portfolio-name" class="title-1">
+            </h2>
```
- [ ] Please notice that the title of the work section should be in a `h2` tag and should not be a image (`img`). this will make your code much more maintainable 😄 (specialy when you will want to change the title)

Lookingfor00 commented on index.html:
```diff
@@ -43,6 +43,179 @@ <h1 class="text1">Hey There.<br>I'm Chris.</h1>
+                </div>
+                <div class="about-section">
+                  <section class="about-grid">
+                  <nav class="about-h1-section">
+                    <h1 class="h1-text">
+                      About me
+                    </h1>
+                  </nav>
```
- [ ] Please do not wrap the h1 element inside a `nav`. The `nav` should be used only to wrap a list of an element that helps to navigate through the website. Also, consider changing `h1` to `h2` as each page should have only one `h1`. This will improve greatly the semantic of your webpage 😄 which will make your website much more accessible (for users, screen readers, and search engines).

Lookingfor00 commented on index.html:
```diff
@@ -43,6 +43,179 @@ <h1 class="text1">Hey There.<br>I'm Chris.</h1>
+                  <button class="sec-grid-item-5"><a href="index.html" class="see-project-t">see project</a></button>
+                </section>
+                </div>
+                <div class="about-section">
+                  <section class="about-grid">
```
- [ ] Please consider aligning your about section with flex-box. as require in this [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) 👍

Lookingfor00 commented on index.html:
```diff
@@ -43,6 +43,179 @@ <h1 class="text1">Hey There.<br>I'm Chris.</h1>
+                  <ul class="sec-grid-item-3">
+                    <li><button class="button"><a href="#" class="but">Html</a></button></li>
+                    <li class="but"><button class="button"><a href="#" class="but">bootstrap</a></button></li>
+                    <li class="but"><button class="button"><a href="#" class="but">Ruby</a></button></li>
+                  </ul>
+                  <button class="sec-grid-item-5"><a href="index.html" class="see-project-t">see project</a></button>
+                </section>
+                </div>
+                <div class="sec-grid-container">
+                  <section class="background">
+                  <h2 class="sec-grid-item-1 text-1">
+                    Profesional Art <br> Printing Data
+                  </h2>
+                  <p class="sec-grid-item-2 text-2">
+                    A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups required. Has been the <br> industry's standard.
+                  </p>
+                  <ul class="sec-grid-item-3">
+                    <li><button class="button"><a href="#" class="but">Html</a></button></li>
+                    <li class="but"><button class="button"><a href="#" class="but">bootstrap</a></button></li>
+                    <li class="but"><button class="button"><a href="#" class="but">Ruby</a></button></li>
+                  </ul>
+                  <button class="sec-grid-item-5"><a href="index.html" class="see-project-t">see project</a></button>
+                </section>
+                </div>
+                <div class="last-grid-container">
+                  <section class="background">
+                  <h2 class="sec-grid-item-1 text-1">
+                    Profesional Art <br> Printing Data
+                  </h2>
+                  <p class="sec-grid-item-2 text-2">
+                    A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups required. Has been the <br> industry's standard.
+                  </p>
+                  <ul class="sec-grid-item-3">
+                    <li><button class="button"><a href="#" class="but">Html</a></button></li>
+                    <li class="but"><button class="button"><a href="#" class="but">bootstrap</a></button></li>
+                    <li class="but"><button class="button"><a href="#" class="but">Ruby</a></button></li>
+                  </ul>
+                  <button class="sec-grid-item-5"><a href="index.html" class="see-project-t">see project</a></button>
+                </section>
+                </div>
```
- [ ] Please consider wrapping all these elements into one work `section`. This will make your code look much cleaner 🧼 and maintainable 😄

Lookingfor00 reviewed (changes requested):
# Require Change ♻️ 
Hi @gisachris, 

Great job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
![almost-there-reach](https://user-images.githubusercontent.com/74411135/225062441-793da0f4-3ef1-4d98-bee9-9c200c7121a7.gif)

## Highlights
- All linters pass 🍏 
- Professional documented project 📖 

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly, add button interaction like `hover` `active` and `enabled`. This will make your project more interactive and user-friendly 📱😄
- [ ] Please use the correct active color for all elements. You are currently using the color `RGB(196, 103, 16);`, please update it to `#FF6B00` (in the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-1471&t=yDKg5dMjRcmmZ2y5-0)). This will make your project look way prettier 🌷 😄 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@Trast00) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

gisachris commented:
after my last code review the following have been added:
added button interaction
changed all to use the correct active color like:(#ffbb45)
changed description of the work element to be in a paragraph and not image
changed body container should use grid 
put only one h1 element in document
wrapped all work to a section (as requested)
change grid-container and sec-grid-container to display flex

gisachris committed:
Update index.html

gisachris committed:
Update styling.css

gisachris committed:
Update styling.css

gisachris committed:
Update styling.css

gisachris committed:
Update styling.css

gisachris committed:
Update styling.css

bobb-Rob reviewed (changes requested):
Hi @gisachris, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has unresolved stylelint issues. Rn this `npx stylelint "**/*.{css,scss}" --fix` to automatically fix almost all of it.
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

gisachris committed:
Update styling.css

gisachris committed:
Update styling.css

gisachris commented:
hello robert
I have quickly fixed all the issues. if you could look at my request again.
I would be very grateful

On Sat, Mar 25, 2023 at 7:34 PM Robertson Arthur ***@***.***>
wrote:

> ***@***.**** requested changes on this pull request.
>
> Hi @gisachris <https://github.com/gisachris>,
>
> While you made a great effort in this project, unfortunately, I cannot
> proceed to review your code.
> Invalid Code Review Request
>
> You have submitted a project, that has unresolved stylelint issues. Rn
> this npx stylelint "**/*.{css,scss}" --fix to automatically fix almost
> all of it.
> Your Code Review Request will be marked as invalid in your Dashboard, so
> please submit a new one once you are ready 🙏
>
> Cheers and Happy coding!👏👏👏
>
> Feel free to leave any questions or comments in the PR thread if something
> is not 100% clear.
>
> *Please, do not open a new Pull Request for re-reviews. You should use the
> same Pull Request submitted for the first review, either valid or invalid
> unless it is requested otherwise.*
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/gisachris/week1_portfolio/pull/2#pullrequestreview-1357904624>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A56BY4532EK6SMJSIKKOO5DW55QGXANCNFSM6AAAAAAWHVADDM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

gisachris committed:
secondary-additions

divinecharlotte commented on index.html:
```diff
@@ -19,7 +20,7 @@
           </header>
       <main class="section2">
```
- [ ] kindly consider checking on this [requested change](https://github.com/gisachris/week1_portfolio/pull/2/files#r1148415692) from the previous reviewer and make sure you fixed it to make your work more professional

divinecharlotte commented on index.html:
```diff
@@ -43,6 +44,180 @@ <h1 class="text1">Hey There.<br>I'm Chris.</h1>
+        </nav>
+      </header>
+      <!-- work and about-->
+        <main class="body-container">
```
kindly consider sticking tho the provided figma design as much as possible
- check the image below  👇 
![lll](https://user-images.githubusercontent.com/60146030/227745142-59ef8d9e-fef7-4679-9f6f-428d0e75cefc.PNG)
consider displaying all the works cards in the middle of the page decrease the padding too and this will make your project look more good

gisachris replied:
since my last CR i have changed the following:
changed last slide issues
make body container display:flex
add button interactions hover,active

mahbubul14 replied:
- [ ] As the previous reviewer suggested, Kindly work on it. If it works properly on your browser you can record a screen video and attach it to your PR. :+1: 
[horizontal scrollbar.webm](https://user-images.githubusercontent.com/70483835/227746919-14ab4c87-ab7b-4636-a3f7-f435f07aa0ae.webm)


> kindly consider sticking tho the provided figma design as much as possible
> 
> * check the image below  point_down
>   ![lll](https://user-images.githubusercontent.com/60146030/227745142-59ef8d9e-fef7-4679-9f6f-428d0e75cefc.PNG)
>   consider displaying all the works cards in the middle of the page decrease the padding too and this will make your project look more good

divinecharlotte reviewed (changes requested):
Hi @gisachris   , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive commit messages ✔️ 
- clean code ✔️ 
- Descriptive PR ✔️ 

![](https://media.giphy.com/media/tKyrc0NTlEcGWdThp4/giphy.gif)

## Required Changes ♻️

- [ ] as requested by the previous reviewer  Kindly consider, adding button interaction like hover active and enabled. This will make your project more interactive and user-friendly 📱😄


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

gisachris committed:
Update styling.css

mahbubul14 reviewed (changes requested):
Hi @gisachris,

Great work on making the changes requested by the previous reviewer 👏

You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

# Required Changes ♻️

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

* N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```
